docs/protobuf.md

```diff
@@ -1,5 +1,19 @@
 # Protocol Buffers
 
+ProtoBuf is a format used both to save data in storage and transmit
+data between services. You can think of it as similar to JSON with
+schemas, given that you can use basic types, list and records. Except
+that it's usually represented in an efficient byteform and not in a
+human readable way.
+
+In this document we'll first explain how to encode and decode protobuf
+messages in the three languages used in anki. Typescript, Python and
```

Comment:
You don't actually talk about encoding/decoding below - you talk about invoking methods.